src/core/Surface.ts

```diff
@@ -4,39 +4,40 @@ import { Container } from './types'
 import { ResourceManager } from '.'
 import { BridgeBrush, BridgeClip } from './ResourceManager'
 import { RenderOp } from './RenderOperation'
+import { NativeApi, N } from './nativeApi'
 
-const native = require('../../native')
+const native: NativeApi = require('../../native')
 
 // container with a layout and an optional brush
 // children should have a layout too
 class Surface implements Container<Surface> {
```

Comment:
this is not 100% correct - Text is not a surface and yet it can be added to a Surface, but we can probably leave as it is, for now